src/Symfony/Bridge/PhpUnit/Legacy/SymfonyTestsListenerTrait.php

```diff
@@ -49,7 +49,7 @@ public function __construct(array $mockedNamespaces = array())
             \PHPUnit_Util_Blacklist::$blacklistedClassNames[__CLASS__] = 2;
         } elseif (method_exists(Blacklist::class, 'addDirectory')) {
             (new BlackList())->getBlacklistedDirectories();
-            Blacklist::addDirectory(\dirname(\dirname((new \ReflectionClass(__CLASS__))->getFileName())));
+            Blacklist::addDirectory(\dirname((new \ReflectionClass(__CLASS__))->getFileName(), 2));
```

Comment:
Could we really use this? The second argument of `dirname()` was added in PHP 7.0.0 (https://www.php.net/dirname) and Symfony 3.4 requires PHP 5.5. (https://github.com/symfony/symfony/blob/3.4/composer.json).

Reply:
If the namespaced class exists and that method too, as we're sure in this if branch, that means that we're on PHPUnit >9.1.2, so we're on PHP 7.2 at least.

Reply:
Cool. Then this is fine! Thanks!